libmscore/element.h

```diff
@@ -234,7 +235,7 @@ class Element : public QObject, public ScoreElement {
             SYSTEM_DIVIDER,
             STEM_SLASH,
             LINE,
-
+            ANNOTATION,
             ARPEGGIO,
             ACCIDENTAL,
```

Comment:
Might be, I updated the PR accordingly. It was/is building fine for me,
maybe this was causing mtest to fail, let's see..

On Sun, Jun 19, 2016 at 7:05 PM, Joachim Schmitz <notifications@github.com>
wrote:

> In libmscore/element.h
> <https://github.com/musescore/MuseScore/pull/2666#discussion_r67617116>:
>
> > @@ -234,7 +235,7 @@ class Element : public QObject, public ScoreElement {
> >              SYSTEM_DIVIDER,
> >              STEM_SLASH,
> >              LINE,
> > -
> > +            ANNOTATION,
> >              ARPEGGIO,
> >              ACCIDENTAL,
>
> I think you'd Need to move ANNOTATION after ACCIDENTAL here, to have the
> same order as in element.cpp, elementNames[]? This is what might be
> causing the mtest failures?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/musescore/MuseScore/pull/2666/files/570f54e182baa7112ea83b42aa1d1db6c6a8dce6#r67617116>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGf7ysNtIIi0Tfahsb0V7Y3m01Pbsashks5qNUWHgaJpZM4I1g-h>
> .
>



-- 
Ruchit Agrawal.
IIIT Hyderabad.